travis/.rubocop_rspec_base.yml

```diff
@@ -2,7 +2,7 @@
 # can customize by inheriting this file and overriding particular settings.
 
 AccessModifierIndentation:
-  EnforcedStyle: outdent
```

Comment:
we weren't consistently respecting this in repos (particularly with `module_function`)